play/nix/shell.nix

```diff
@@ -14,14 +14,17 @@ stdenv.mkDerivation rec {
   buildInputs = with pkgs; [
       cmake
       rustup
+      openssl
+      pkg-config
       lld
       python38Packages.pip
     ];
 
   hardeningDisable = [ "fortify" ];
 
+  MZ_DEV = 1;
+  RUSTFLAGS = "-C link-arg=-fuse-ld=lld -C debuginfo=0";
   shellHook = ''
-    export MZ_DEV=1
-    export RUSTFLAGS="-C link-arg=-fuse-ld=lld -C debuginfo=0"
+    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${lib.makeLibraryPath buildInputs}"
```

Comment:
Weird that this should be necessary. Maybe openssl should go on `nativeBuildInputs`? (I usually just experiment with those until I hit the one that sets the right env vars)

Reply:
No, it's a normal build input because we link to it. I think the problem is that normally ELFs need to be patched to get their dynamic libs working but inside a nix shell you're just invoking cargo

Reply:
To avoid spending more time on this I'll go ahead and merge it, but feel free to change it if you find a better way